src/main/java/org/killbill/billing/plugin/analytics/api/core/AnalyticsConfiguration.java

```diff
@@ -19,47 +19,88 @@
 
 package org.killbill.billing.plugin.analytics.api.core;
 
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Properties;
+
+import org.killbill.billing.plugin.analytics.AnalyticsActivator;
 
 import com.google.common.collect.ImmutableMap;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 @SuppressFBWarnings("UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD")
 public class AnalyticsConfiguration {
 
-    // Sane defaults (keys used by most official plugins)
-    private final Map<Integer, String> defaultPluginPropertyKeys = ImmutableMap.<Integer, String>of(1, "processorResponse",
-                                                                                                    2, "avsResultCode",
-                                                                                                    3, "cvvResultCode",
-                                                                                                    4, "payment_processor_account_id",
-                                                                                                    5, "paymentMethod");
-
     // List of account ids to ignore
-    public List<String> blacklist = new LinkedList<String>();
+    public final List<String> blacklist;
```

Comment:
Were you able to confirm the YAML-based configuration still works?

IIRC it wasn't `final` because reflection is used when loading/parsing the YAML content (see `UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD` above). I guess now the default no-arg constructor is used? If so, I think you can removed the `SuppressFBWarnings` annotation.

Reply:
>Were you able to confirm the YAML-based configuration still works?

Yes, I've tested the code, the YAML based configuration also works.  I've now removed the `SuppressFBWarnings` annotation.